interns/tamara-nicole-medrano/journals/index.html

```diff
@@ -13,6 +13,23 @@ <h1>Journals</h1>
     width="800px"
     height="500px"> 
   </object>
+
+ <h2>Journals</h2>
+ 
+  <object
+    data="./Tamara Medrano - Week 2 Journal Entries .pdf"
+    width="800px"
+    height="500px"> 
+  </object>
+
+ <h3>Journals</h3>
  
+  <object
+    data="./Tamara Medrano - Week 3 Journal Entries .pdf"
+    width="800px"
+    height="500px"> 
+  </object>
```

Comment:
I remember that we had issues with having a space before `.pdf`. Please rename your file from Finder by removing the space and then remove the extra space in in your PR